src/components/results/response/style.css

```diff
@@ -0,0 +1,25 @@
+.response__json-view {
+    width: 100%;
+    height: 100%;
+}
+
+.response {
+    margin-top: 12px;
+}
+
+.response__copy-to-clipboard {
+    background-color: #2487bc;
+    color: white;
+    display: block;
+    margin: 0;
+    height: 24px;
+    width: 128px;
+    border-width: 0;
+    border-width: 1px;
```

Comment:
more duplicated CSS. we should remove these so that we don't confuse people in the future. "why is `border-width: 0` there? It must be there for a reason!!?"